.github/workflows/py3_casacore_master.docker

```diff
@@ -14,6 +14,7 @@ ADD . /src
 WORKDIR /src
 RUN python3 -m venv venv && \
     . venv/bin/activate && \
+    pip install numpy==1.26.4 && \
```

Comment:
Would it not be safer to use `numpy<2`, or do you really need this pinned version? I'm not sure 1.26.4 is available for all Python versions present in Ubuntu 20.04, 22.04 and 24.04 (well the latter should be fine :))

Reply:
Agree, `<2` is exactly what I meant. Changed now.